docs/first-order/modality_alignment.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>replace "evoked" with "co-occuring"</p><p>Causality is very hard to support without more extensive analysis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>